.git2gus/config.json

```diff
@@ -20,7 +20,7 @@
     "area:aura": "a1aB0000000MRAzIAO",
     "area:lwc": "a1aB0000000Bn8tIAC",
     "area:slds": "a1aB000000005G3IAI",
```

Comment:
can we take out this area please, slds should not create work items with IDEx product tag